### PR TITLE
Update sysdig notification doc

### PR DESCRIPTION
**Pull request by w8896699**

### CODE REVIEW

File: src/docs/app-monitoring/sysdig-monitor-create-alert-channels.md [hunk 1]
```diff
@@ -25,14 +25,18 @@ You can create alerts based on monitoring dashboards in Sysdig Monitor, that not
 Here are some steps on how to setup the Sysdig alerts with [Rocket.Chat](https://chat.developer.gov.bc.ca/).
 
 ## On this page
-- [Create a Rocket.Chat chat channel and webhook for alert messages](#create-a-rocketchat-chat-channel-and-webhook-for-alert-messages)
-- [Create a Sysdig team notification channel](#create-a-sysdig-team-notification-channel)
-- [Creating an alert](#creating-an-alert)
+- [Create alerts and notifications in Sysdig Monitor](#create-alerts-and-notifications-in-sysdig-monitor)
+  - [On this page](#on-this-page)
```
Comment (ShellyXueHan) [inline src/docs/app-monitoring/sysdig-monitor-create-alert-channels.md:29]:
```suggestion
```

**Review by ShellyXueHan — APPROVED** [review]

**Comment by ShellyXueHan** [comment]
@w8896699 please also ask Pilar to review documentations!

**Comment by Pilargit12** [comment]
I'm working on reviewing it today and making changes. Thank you Team!

**Review by Pilargit12 — APPROVED** [review]
I've made some minor spelling changes, added last reviewed and web-style guideline changes too.

**Pull request merged**